spec/factories/workflow_steps.rb

```diff
@@ -15,5 +15,11 @@
       status { 'completed' }
       completed_at { Time.now }
     end
+
+    trait :with_ocr_metadata do
+      after(:create) do |step|
+        create(:version_metadata, druid: step.druid, version: step.version)
+      end
+    end
```

Comment:
Since we moved away from hanging this info off the workflow step model, I wonder if we might also consider moving this factory (or removing it since you created a version metadata factory)?

Reply:
this is allowing me to use the version metadata factory to create some metadata for the step all at once, which is useful for tests where i am verifying that steps with metadata work as expected, e.g. like in this test: https://github.com/sul-dlss/workflow-server-rails/blob/workflow-variables/spec/requests/workflows/all_workflows_for_one_object_spec.rb#L55-L75

Reply:
Sounds good.